alf/trainers/policy_trainer.py

```diff
@@ -694,7 +695,8 @@ def play(root_dir,
             time_step = env.reset()
 
     for m in metrics:
-        logging.info("%s: %f", m.name, m.result())
+        logging.info("%s: %s", m.name,
+                     map_structure(lambda x: x.numpy().item(), m.result()))
```

Comment:
Do we need item()? It could be an np.array for each element?

Reply:
``item()`` is needed, as the ``AverageReturnMetric`` returns a list of dim-0 tensors (one for each return/reward dimension) in the case of play.
Added support to the case of np.array for each element, although currently there is no such metrics in play.